app/views/spam2/_insights.html.erb

```diff
@@ -0,0 +1,214 @@
+<div class="card-deck w-100 mb-3">
+  <div class="card">
+    <div class="card-body">
+    <h5 class="card-title  text-secondary font-weight-bold"> <i class="fa fa-circle text-primary"></i>  Unmoderated</h5>
+      <ul class="list-group list-group-flush text-secondary ">
+        <li class="list-group-item"><%= Node.where(status: 4).count%> Nodes</li>
+        <li class="list-group-item"><%= Comment.where(status: 4).count %> Comments</li>
+      </ul>
+    </div>
+  </div>
+  <div class="card">
+   <div class="card-body">
+       <h5 class="card-title  text-secondary font-weight-bold"> <i class="fa fa-ban text-danger"></i>  Spammed</h5>
+      <ul class="list-group list-group-flush text-secondary ">
+        <li class="list-group-item"><%= Node.where(status: 0).count%> Nodes</li>
+        <li class="list-group-item"><%= Comment.where(status: 0).count %> Comments</li>
+      </ul>
+    </div>
+  </div>
+  <div class="card">
+    <div class="card-body">
+        <h5 class="card-title  text-secondary font-weight-bold"> <i class="fa fa-flag text-warning"></i>  Flagged</h5>
+      <ul class="list-group list-group-flush text-secondary ">
+        <li class="list-group-item"><%= Node.where('flag > ?', 0).count%> Nodes</li>
+        <li class="list-group-item"><%= Comment.where('flag > ?', 0).count %> Comments</li>
+      </ul>
+    </div>
+  </div>
+    <div class="card">
+    <div class="card-body">
+        <h5 class="card-title  text-secondary font-weight-bold"> <i class="fa fa-check text-success"></i>  Published</h5>
+        <ul class="list-group list-group-flush text-secondary ">
+            <li class="list-group-item"><%= Node.where(status: 1).count%> Nodes</li>
+            <li class="list-group-item"><%= Comment.where(status: 1).count %> Comments</li>
+        </ul>
+    </div>
+  </div>
+</div>
+
+<div id="timeline" class="mt-5 w-100"></div>
+
+<div class="card-deck w-100 mt-5">
+  <div class="card border-0">
+   <div class="card-body">
+    <h5 class="card-title  text-secondary font-weight-bold"> <i class="fa fa-tags text-primary"></i> Tags followed by moderators and admins</h5>
+    <% if @popular_tags.present? %>
```

Comment:
Maybe use `.exists?` instead of .`present?` reference https://semaphoreci.com/blog/2017/03/14/faster-rails-how-to-check-if-a-record-exists.html also sidenote: @cesswairimu @jywarren should we do a refactoring for this in the app too?